partiql-planner/src/main/kotlin/org/partiql/planner/internal/typer/PlanTyper.kt

```diff
@@ -1218,8 +1218,8 @@ internal class PlanTyper(
         fun resolveAgg(agg: Agg.Unresolved, arguments: List<Rex>): Pair<Rel.Op.Aggregate.Call, StaticType> {
             var missingArg = false
             val args = arguments.map {
-                val arg = visitRex(it, null)
-                if (arg.type.isMissable()) missingArg = true
+                val arg = visitRex(it, it.type)
+                if (arg.type is MissingType) missingArg = true
```

Comment:
I'm wondering why the `missingArg` matters now. Aggregates filter out all null/missing afaik so we kind of ignore them in typing?

Reply:
Yeah, that makes sense to me. Removed.